frontend/src/scenes/billing/billingLogic.tsx

```diff
@@ -83,6 +91,8 @@ export const billingLogic = kea<billingLogicType>([
             ['resetDismissKey as resetUsageLimitExceededKey'],
             lemonBannerLogic({ dismissKey: 'usage-limit-approaching' }),
             ['resetDismissKey as resetUsageLimitApproachingKey'],
+            supportLogic,
+            ['openSupportForm'],
```

Comment:
Thanks for reminding. Was harebrained and forgot I was trying to make this work.

Reply:
This isn't possible the way I currently set it up so am going to revert this addition. The problem is that deactivateProduct is on billingLogic but unsub survey information is on billingProductLogic. I think I'll refactor this in a subsequent PR so that deactivateProduct lives on billingProductLogic instead.